src/store/models/taro.ts

```diff
@@ -43,10 +53,13 @@ export interface TaroModel {
   getAssets: Thunk<TaroModel, TaroNode, StoreInjections, RootModel>;
   setBalances: Action<TaroModel, { node: TaroNode; balances: PTARO.TaroBalance[] }>;
   getBalances: Thunk<TaroModel, TaroNode, StoreInjections, RootModel>;
-  getNewAddress: Thunk<TaroModel, NewAddressPayload, StoreInjections, RootModel>;
+
   getAllInfo: Thunk<TaroModel, TaroNode, RootModel>;
   mineListener: ThunkOn<TaroModel, StoreInjections, RootModel>;
   mintAsset: Thunk<TaroModel, MintAssetPayload, StoreInjections, RootModel>;
+  getNewAddress: Thunk<TaroModel, NewAddressPayload, StoreInjections, RootModel>;
+  sendAsset: Thunk<TaroModel, SendAssetPayload, StoreInjections, RootModel>;
+  decodeAddress: Thunk<TaroModel, DecodeAddressPayload, StoreInjections, RootModel>;
 }
```

Comment:
I noticed that the return type of `decodeAddress` call in `SendAssetModal` was `any`. It would be helpful to update this so callers get the proper return type.

```ts
  decodeAddress: Thunk<
    TaroModel,
    DecodeAddressPayload,
    StoreInjections,
    RootModel,
    Promise<PTARO.TaroAddress>
  >;
```

It would make sense to update `getNewAddress` as well.

Reply:
Done. I dont understand this too well, I'm going to read up on thunks. It looks like the last argument is the result?